packages/reports/tests/integration/components/navi-date-picker-test.js

```diff
@@ -89,6 +89,26 @@ module('Integration | Component | Navi Date Picker', function(hooks) {
     assert.ok(!isDayActive(clickDate), 'Clicked date is no longer selected');
   });
 
+  test('Change center date', async function(assert) {
+    assert.expect(4);
+
+    const getMonth = () => find('.ember-power-calendar-nav-title > span').childNodes[2].textContent;
+    this.set('date', moment('2015-07-14', TEST_FORMAT));
+
+    await render(hbs`<NaviDatePicker @date={{this.date}} /> `);
+
+    assert.equal(getMonth(), 'July', 'The center date starts as the passed in date');
+
+    await click('.ember-power-calendar-nav-control--previous');
+    assert.equal(getMonth(), 'June', 'The center date changes to previous month');
+
+    await click('.ember-power-calendar-nav-control--next');
+    assert.equal(getMonth(), 'July', 'The center date goes back to original after click');
+
+    await click('.ember-power-calendar-nav-control--next');
+    assert.equal(getMonth(), 'August', 'The center date goes one month ahead');
```

Comment:
Do we test changing the month through clicking the month name and selecting a month from the list?

Reply:
Nope

Reply:
@mattgraham1995 https://stackoverflow.com/q/430237? Doesn't look like it's easily done unless we switch to basic dropdown/etc

Reply:
It wouldn't even work in an acceptance test?

Reply:
Nope, can't trigger it to open, but I was able to do manually change the selected item and then trigger a `'change'` event

```js
const monthSelect = getMonthSelect();
monthSelect.selectedIndex = monthSelect.selectedIndex - 1;
await triggerEvent(monthSelect, 'change');
```